rpi_backlight/__init__.py

```diff
@@ -160,13 +193,55 @@ def power(self) -> bool:
         :setter: Set the display power on or off.
         :type: bool
         """
-        # 0 is on, 1 is off
-        return not self._get_value("bl_power")
+
+        if self._board_type == BoardType.RASPBERRY_PI:
+            # 0 is on, 1 is off
+            return not self._get_value("bl_power")
+        elif self._board_type == BoardType.TINKER_BOARD:
+            if get_brightness_value() == 0:
+                value = 255
+            else:
+                value = 0
+            # if smooth:
+            #     if not isinstance(duration, (int, float)):
+            #         raise ValueError(
+            #             "integer or float required, got '{}'".format(type(duration))
+            #         )
+            #     actual = get_brightness_value()
+            #     diff = abs(value - actual)
+            #     while actual != value:
+            #         actual = actual - 1 if actual > value else actual + 1
+            #         _set_brightness_value(actual)
+            #         time.sleep(duration / diff)
+            # else:
+            self._set_value("tinker_mcu_bl", value)
+
+
 
     @power.setter
     def power(self, on: bool) -> None:
         """Set the display power on or off."""
         if not isinstance(on, bool):
             raise TypeError("value must be a bool, got {0}".format(type(on)))
-        # 0 is on, 1 is off
-        self._set_value("bl_power", int(not on))
+        if self._board_type == BoardType.RASPBERRY_PI:
+            # 0 is on, 1 is off
+            self._set_value("bl_power", int(not on))
+        elif self._board_type == BoardType.TINKER_BOARD:
+            if on:
+                value = 255
+            else: 
+                value = 0
+            # if smooth:
+            #     if not isinstance(duration, (int, float)):
+            #         raise ValueError(
+            #             "integer or float required, got '{}'".format(type(duration))
+            #         )
+            #     actual = get_brightness_value()
+            #     diff = abs(value - actual)
+            #     while actual != value:
+            #         actual = actual - 1 if actual > value else actual + 1
+            #         self._set_value("tinker_mcu_bl", actual)
+            #         time.sleep(duration / diff)
+            # else:
+            #     self._set_value("tinker_mcu_bl" ,value)
+            self._set_value("tinker_mcu_bl", value)
```

Comment:
Setting `value` like this seems overly verbose, let's just do

```suggestion
            self._set_value("tinker_mcu_bl", 255 if on else 0)
```

Reply:
I didnt know this shortcut. Thanks!